lib/generators/format/format_generator.rb

```diff
@@ -0,0 +1,20 @@
+require 'json'
+
+class FormatGenerator < Rails::Generators::NamedBase
+  source_root File.expand_path('../templates', __FILE__)
+
+  def copy_format_files
+    format_name = file_name
```

Comment:
It'd be good to call `underscore` on this. Rails generators for e.g. model accept both forms and turn them into the correct thing, e.g. `rails g model FooBar` is equivalent to `rails g model foo_bar`.